src/Table.tsx

```diff
@@ -12,6 +12,8 @@ interface Props<RowShape> {
 
   density?: "standard" | "condensed" | "relaxed";
 
+  loading?: boolean;
```

Comment:
I don't think we should put this into space kit right now. First, I think the current tables implementation would allow you to show the exact same loading state as you have here because the content that's output is totally configurable, meaning you can configure it to show this loading state.

I also would like feedback from @adamatorres regarding this loading state. I'd be happy to settle for a loading spinner showing before we see this table at all like we do in other parts of the application instead of coming up with a unique loader that we're going to remove later.

Reply:
I would guess loading bars are the preferred loading state, and that should definitely be possible to implement in engine rather than here as Justin mentioned. +1 for doing this in engine rather than space kit